DB/api/url_parser.py

```diff
@@ -47,4 +55,5 @@ def parse(url):
 	if not query_options['device_id']:
 		raise Exception("Not Found")
 
+	print(query_options)
```

Comment:
Can we make this more of a logging statement? At the very minimum, add a title. Ex. print("query options:", query_options)